### PR TITLE
Update to 0.22 deps

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by dsimansk** [comment]
Please bump the version in test as well, https://github.com/knative-sandbox/kn-plugin-source-kafka/blob/main/test/e2e-tests.sh#L26-L27 and L45.

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -23,8 +23,8 @@ export PATH=$PATH:${REPO_ROOT_DIR}
 # Will create and delete this namespace (used for all tests, modify if you want a different one used)
 export KN_E2E_NAMESPACE=kne2etests
 
-export KNATIVE_EVENTING_VERSION="0.21.0"
-export KNATIVE_SERVING_VERSION="0.21.0"
+export KNATIVE_EVENTING_VERSION="0.22.0"
+export KNATIVE_SERVING_VERSION="0.22.0"
```
Comment (dsimansk) [inline test/e2e-tests.sh:27]: And L45 pls, I can't make a suggestion for it :)
https://github.com/knative-sandbox/kn-plugin-source-kafka/blob/main/test/e2e-tests.sh#L45

Comment (rhuss) [reply test/e2e-tests.sh:27]: thx! Should be fixed now

**Review by dsimansk — APPROVED** [review]
Thanks!
/lgtm

**Pull request merged**